californium-core/src/main/java/org/eclipse/californium/core/coap/CoAP.java

```diff
@@ -408,6 +408,7 @@ public static Code valueOf(final int value) {
 				case 4: return DELETE;
 				case 5: return FETCH;
 				case 6: return PATCH;
+
```

Comment:
Please, try to omit changing files only for formatting.

Reply:
I will remove this.